src/components/LoginForm/FormFieldsHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import Image from "next/image";
+import { twMerge } from "tailwind-merge";
+
+import { Flex } from "@aws-amplify/ui-react";
+
+export default function FormFieldsHeader({
+  className,
+}: {
+  className?: string;
+}) {
+  return (
+    <div
+      className={twMerge(
+        "relative ml-4 mt-4 flex flex-col justify-center bg-white px-8" +
+          className,
+      )}
+    >
+      <Image
+        className="absolute -right-28 top-0"
+        src={"/svgs/login/Star_Icon.svg"}
+        height={51}
+        width={59}
+        alt={""}
+      />{" "}
+      <Image
+        className="absolute -left-28 bottom-0"
+        src={"/svgs/login/Star_Icon.svg"}
+        height={51}
+        width={59}
+        alt={""}
+      />
+      <div className="text-lg font-semibold md:text-2xl xl:text-4xl">
+        <Flex direction={"column"}>
+          <Flex>
+            <Image
+              className=" pointer-events-none -mr-4 select-none"
+              src="/svgs/login/vector_112.svg"
+              alt=""
+              width={28}
+              height={21}
+            />
+            <div className=" text-nowrap">Individual Registration</div>
+          </Flex>
```

Comment:
For some reason I can't see this on Safari unless I add this
```suggestion
            <div className="text-nowrap text-black">
```